NotadogApi/Domain/Game/Room.cs

```diff
@@ -63,6 +66,13 @@ public void changeState(IRoomState state)
         public void addPlayer(User user)
         {
             Players.Add(user);
+
+            if (PlayersMaxCount > 0 && PlayersMaxCount == Players.Count)
+            {
+                changeState(new WaitingStartState(this));
```

Comment:
PlayingState?

Reply:
nope, if we have maxCount we should start after some time